src/olympia/shelves/tests/test_serializers.py

```diff
@@ -86,85 +85,64 @@ def setUp(self):
             api_settings.DEFAULT_VERSIONING_CLASS()
         )
         self.request.version = api_version
-        self.request.user = AnonymousUser()
+        self.request.user = user_factory()
```

Comment:
I missed this - is it a new change?  Does AnonymousUser no longer work?  (the API should work when not logged in)

Reply:
AnonymousUser works, I'll change it back.